packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -75,11 +76,15 @@ function createScale(scale: Scale | null, domain: VDomain | undefined): Scale {
 
   const _defaultScale = domain ? domain.value.toDefaultScale() : defaultScale;
 
-  // This gets min/max from domain, if it's not on scale.
-  return {
-    ..._defaultScale,
-    ...(scale || {}),
-  };
+  // _defaultScale can have a lot of undefined values. These should be over-written.
+  const resultScale = mergeWith(
+    scale || {},
```

Comment:
```suggestion
    {},
    scale || {},
```

Otherwise, the original scale object will be modified (`mergeWith` is mutating).

Example how this is a problem: https://squiggle-website-git-plot-with-pa-19054e-quantified-uncertainty.vercel.app/playground?v=dev#code=eNqrVkpJTUsszSlxzk9JVbJSKlawVQhOTsxJ1SuuzM3JT9eoVkjOzysuScwrsVIwVqjVjMkrACoJyMkv0csrzU0tykx2ywMqSsuzUqiuqbBSiDbVUTCLrVGoqNVRqACbZKVQDNFnhEcjNg1KtQB3CzS3

Reply:
Eek, good catch!